docs/BigQueryTable-batchsource.md

```diff
@@ -55,6 +55,17 @@ It will be automatically created if it does not exist, but will not be automatic
 Temporary data will be deleted after it has been read. If it is not provided, a unique bucket will be
 created and then deleted after the run finishes.
 
+**Enable Querying Views**: Whether to allow querying views. Since BigQuery views are not materialized 
```

Comment:
maybe this need to be like this: "Enable Querying view" , in documentation is like this

Reply:
I agree for **Enable Querying Views**

I don't know if we need the second part:

```
Since BigQuery views are not materialized 
```